Intermittent NullPointerException

Travis CI initially failed in [PR 110](https://github.com/ScorexFoundation/Scorex/pull/110), because a NullPointerException was thrown during the execution of the test "NodeViewHolder: forking - switching" in NodeViewHolderTests.

```
[info] - NodeViewHolder: forking - switching *** FAILED ***
[info]   java.lang.NullPointerException:
[info]   at scorex.testkit.properties.NodeViewHolderTests.$anonfun$$init$$39(NodeViewHolderTests.scala:239)
[info]   at scorex.testkit.properties.NodeViewHolderTests.$anonfun$$init$$39$adapted(NodeViewHolderTests.scala:211)
[info]   at scorex.testkit.properties.NodeViewHolderTests.withFixture(NodeViewHolderTests.scala:47)
[info]   at scorex.testkit.properties.NodeViewHolderTests.$anonfun$$init$$38(NodeViewHolderTests.scala:211)
[info]   at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
[info]   at org.scalatest.OutcomeOf.outcomeOf(OutcomeOf.scala:85)
[info]   at org.scalatest.OutcomeOf.outcomeOf$(OutcomeOf.scala:83)
[info]   at org.scalatest.OutcomeOf$.outcomeOf(OutcomeOf.scala:104)
[info]   at org.scalatest.Transformer.apply(Transformer.scala:22)
[info]   at org.scalatest.Transformer.apply(Transformer.scala:20)
[info]   ...
```

However, when I asked Travis to re-run this build, the test passed.  So, whatever is causing this problem doesn't happen all the time.

Another one:

```
[info] - NodeViewHolder: simple forking *** FAILED ***
[info]   java.lang.AssertionError: assertion failed: timeout (3 seconds) during expectMsgClass waiting for class scorex.core.NodeViewHolder$SyntacticallySuccessfulModifier
[info]   at scala.Predef$.assert(Predef.scala:219)
[info]   at akka.testkit.TestKitBase.expectMsgClass_internal(TestKit.scala:472)
[info]   at akka.testkit.TestKitBase.expectMsgType(TestKit.scala:445)
[info]   at akka.testkit.TestKitBase.expectMsgType$(TestKit.scala:445)
[info]   at akka.testkit.TestKit.expectMsgType(TestKit.scala:814)
[info]   at scorex.testkit.properties.NodeViewHolderTests.$anonfun$$init$$32(NodeViewHolderTests.scala:199)
[info]   at scorex.testkit.properties.NodeViewHolderTests.$anonfun$$init$$32$adapted(NodeViewHolderTests.scala:173)
[info]   at scorex.testkit.properties.NodeViewHolderTests.withFixture(NodeViewHolderTests.scala:47)
[info]   at scorex.testkit.properties.NodeViewHolderTests.$anonfun$$init$$31(NodeViewHolderTests.scala:173)
[info]   at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
```

--- a/examples/src/test/scala/hybrid/ModifierGenerators.scala
+++ b/examples/src/test/scala/hybrid/ModifierGenerators.scala
@@ -40,7 +40,9 @@ trait ModifierGenerators {
     assert(insPerTx >= 1 && insPerTx <= 10)
 
     def filterOutForgedBoxes(in: (ByteArrayWrapper, ByteArrayWrapper)): Boolean = {
-      PublicKey25519NoncedBoxSerializer.parseBytes(in._2.data).map(_.value).getOrElse(0L) > 0
+      //current problem with unstable nodeviewholder spec is caused by coinbase block which always has value 1
+      //so for now we just won't use it
+      PublicKey25519NoncedBoxSerializer.parseBytes(in._2.data).map(_.value).getOrElse(0L) > 1L
     }
 
     val stateBoxes = state.store.getAll()

